src/app/pages/IdxPage/testHelpers/index.jsx

```diff
@@ -8,8 +9,12 @@ import IdxPage from '#pages/IdxPage';
 
 const radioServiceOverride = 'dari';
 
-// eslint-disable-next-line react/prop-types
-const IdxPageWithContext = ({ service = 'persian', pageData }) => (
+/* eslint-disable react/prop-types */
+const IdxPageWithContext = ({
+  service = 'persian',
+  pageData,
+  mostReadEndpointOverride,
```

Comment:
I think we can remove this prop here entirely, and just keep `getLocalMostReadEndpoint({ service })` as our `mostReadEndpointOverride`, this way we are not excessively passing props around.

Then you'd also have to update the [fetchMock url](https://github.com/bbc/simorgh/pull/6872/files#diff-b2d18c3c50e2d6071fe67aa0ef80154dR20) to match whatever `getLocalMostReadEndpoint({ service })`  returns. (in the case of our test above, I think it would be "./data/persian/mostRead/index.json")